### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by nvdaes**
## Link to issue number:
Issue #26 

### Summary of the issue:
Documentation may not be clear, especially for new users.

### Description of how this pull request fixes the issue:
- Change the title of Preferences menu, reflecting that a settings panel is used.
- Describe each control of the panel in a bulletted paragraph.
- Add a note about emulate copy and cut features.
### Testing performed:
Read the changes in documentation.
### Known issues with pull request:
None
### Change log entry:
* Improved documentation.

### CODE REVIEW

**Comment by nvdaes** [comment]
@CyrilleB79 , if you can, please review this and provide feedback. Thanks

File: readme.md
Comment (CyrilleB79) [inline readme.md:33]: The category in documentation was not the same as the one in the code.

```suggestion
*	The above command can be changed from NVDA menu, Preferences submenu, Input gestures dialog, Text review category.
```

Note: Personally, I do not think it is a good idea to have these commands in the Text review category since they can be applied not only in for text review, i.e. in the review cursor context. I would have put them in a category dedicated to this add-on or in the "System status" category, along with "Reports the text on the Windows clipboard".
But this review is focused on the doc, so my suggestion intend to make the doc match the UI.

File: readme.md
Comment (CyrilleB79) [inline readme.md:28]: I still do not understand the difference between the two HTML options. Maybe a sublist with a more detailed description of each option would be clearer? Sorry if I missed something obvious.

Additional note: Also maybe it is a bit off-topic for this PR. But I would have expected the raw text to be the first option, in the doc but also in the UI, since it seems to me the most common use case, i.e. someone wanting to review the content of the clipboard without any interpretation. And also raw text to be the default option (if not already).

**Review by CyrilleB79 — CHANGES_REQUESTED** [review]
I have reviewed the changes and made my comments/suggestions. Also some comments are going a bit further than documentation. Feel free to ignore it in this PR. You may open new PRs if you feel they are appropriated.

File: readme.md
Comment (nvdaes) [reply readme.md:33]: Thanks for your review. About category command to show the add-on settings, currently it is not the same than the other add-on commands. In fact scriptCategory is text review, but this particular command is assigned to config category. I'll create a dedicated category for this add-on, following your suggestion. The code used for this command is:

```
		description=_("Shows the Clip Contents Designer settings."),
		category=SCRCAT_CONFIG
```

File: readme.md
Comment (nvdaes) [reply readme.md:28]: I'll improve the description to explain the difference between preformatted and HTML as shown in a browser. The difference is that preformatted HTML shows spaces and line breaks, while HTML as shown in a browser compacts several spaces in a unique blank character. This is explained in [HTML pre tag - W3Schoolswww.w3schools.com › tags › tag_pre ](https://www.w3schools.com/tags/tag_pre.asp).
Imo, the most useful option for this add-on is the preformatted text, since, if we want to show the raw text, we can more easily paste the text in an editor like Notepad, but we can use the capabilities of NVDA with this add-on to show how the HTML code is rendered, what may not be very easy to see in other ways.

**Comment by nvdaes** [comment]
@CyrilleB79 : I've followed your suggestions adding a new category for the add-on in this same pull request, so we can consider the context of this conversation.
I've explained the difference between "conventional" and preformatted HTML without using a nested list to preserve the format of the documentation, since there aren't nested list for other options. Feel free to provide feedback. If all is OK, I'll merge this and may publish pre-release 13.0-dev.
Thanks a lot!

File: readme.md
Comment (CyrilleB79) [reply readme.md]: > Imo, the most useful option for this add-on is the preformatted text, since, if we want to show the raw text, we can more easily paste the text in an editor like Notepad, but we can use the capabilities of NVDA with this add-on to show how the HTML code is rendered, what may not be very easy to see in other ways.

I have a complete opposite reasoning from you:
* the raw text seems to me the most useful option for a wider range of users because only few users play with HTML code. The use case with raw text is to copy 3 pieces of text from 3 various places and to be able to paste all together at the same place. That's the reason why I have downloaded this add-on. Of course, I may use notepad as I did before, but this add-on allows me to spare a lot of keystrokes to perform frequently such multiple-copy/paste actions.
* On the contrary, when I code some HTML, I have the source code in the editor and the result opened in my browser. If I make some slight change in the source code, I just have to Alt+Tab to my browser and press F5 to see the result.

Now that I have explained with more details my use cases, I'll let you the final decision. You are the add-on author after all. And this is just a matter of default option and order in the documentation. That's not so much!

**Comment by nvdaes** [comment]
Hi, your opinion seams reasonable. I'll think about it and will make a decission. If raw text is set as the defaul option, I'll request you a review in case you have time to test it.
Thanks so much!

**Comment by CyrilleB79** [comment]
Thinking again to it, it seems to me a bit artificial to map these two unrelated use cases to the same shortcut just because they use common code in the implementation. A user may also want to use both use cases frequently and it would be very annoyed to have to change the option in the setting panel each time. Why not have 2 shortcuts:
* one to view the raw text in the clipboard
* one to view the clipboard content as rendered in HTML
The formatting option in the settings panel would then remain with only 2 choices.

**Comment by nvdaes** [comment]
> 
> 
> Thinking again to it, it seems to me a bit artificial to map these two unrelated use cases to the same shortcut just because they use common code in the implementation. A user may also want to use both use cases frequently and it would be very annoyed to have to change the option in the setting panel each time. Why not have 2 shortcuts:
> 
>     * one to view the raw text in the clipboard
> 
>     * one to view the clipboard content as rendered in HTML
>       The formatting option in the settings panel would then remain with only 2 choices.

I think this is an excellent idea. It's implemented now. Please feel free to review changes.
Thanks.

**Comment by CyrilleB79** [comment]
Globally, changes seem OK to me. The doc is OK.

I would just suggest you to update formatLabel string in the GUI as follow to reflect the change you made in last commit:
`formatLabel = _("&Format to show the clipboard text as HTML in browse mode:")`

Sorry, I did not know how to make directly a code suggestion in GitHub; I do not have a "comment" button on this line that is too far from your modifications.

**Comment by nvdaes** [comment]
> 
> 
> Globally, changes seem OK to me. The doc is OK.
> 
> I would just suggest you to update formatLabel string in the GUI as follow to reflect the change you made in last commit:
> `formatLabel = _("&Format to show the clipboard text as HTML in browse mode:")`
> 
> Sorry, I did not know how to make directly a code suggestion in GitHub; I do not have a "comment" button on this line that is too far from your modifications.

I have updated formatLabel as suggested by you, so I'll merge this PR if all checks pass, to release the 15.0-dev version for wide testing.
Thanks.

**Pull request merged**
